File: Topic Based Problems/01 Fundamentals/1 Beginner/15 Calculate Percentage/Java/CalculatePercentage.java
import java.util.Scanner;

public class CalculatePercentage {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double number, totalNumber;

        System.out.print("Enter number: ");
        number = scanner.nextDouble();

        System.out.print("Enter total number: ");
        totalNumber = scanner.nextDouble();

        System.out.printf("Percentage of %.2f is: %.2f%%\n", number, (number / totalNumber) * 100);

        scanner.close();
    }
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/15 Calculate Percentage/C++/CalculatePercentage.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double number, totalNumber;

    cout << "Enter number: ";
    cin >> number;

    cout << "Enter total number: ";
    cin >> totalNumber;

    cout << "Percentage of " << number << " is: " << fixed << setprecision(2) << (number / totalNumber) * 100 << "%" << endl;

    return 0;
}